tests/framework/installer/ceph_installer.go

```diff
@@ -197,18 +197,17 @@ func (h *CephInstaller) CreateK8sRookClusterWithHostPathAndDevices(namespace, sy
 	}
 
 	// Skip this step since the helm chart already includes the roles and bindings
-	if !h.useHelm {
```

Comment:
I was wrong about removing this check for `useHelm`. Looks like it is actually needed for the helm tests to pass the CI.